frontend/src/main/components/BasicCourseSearch/CourseByInstructorSearchForm.js

```diff
@@ -66,7 +66,7 @@ const CourseByInstructorSearchForm = ({ fetchJSON }) => {
             />
           </Col>
           <Form.Group controlId="CourseByInstructorSearch.Instructor">
-            <Form.Label>Instructor (Try searching 'Conrad' or 'CONRAD P T')</Form.Label>
+            <Form.Label>Instructor (Try searching 'MIRZA D' or 'CONRAD P T')</Form.Label>
```

Comment:
The point of this was to let the user know that they could type in `Conrad` and it should work since it ignores case. I think changing it back here (and in the other instances) would be ideal.